docs/concepts/executor/add-endpoints.md

```diff
@@ -428,10 +179,8 @@ class MyExec(Executor):
         return {'internal_parameter': 20}
 
 
-f = Flow().add(uses=MyExec, name='my_executor')
-
-with f:
-    print(f.post(on='/status').to_dict()["parameters"])
+with Deployment(uses=MyExec) as dep:
+    print(dep.post(on='/status').to_dict()["parameters"])
```

Comment:
are we sure this example works out of the box? I doubt so

Reply:
I believe u need `return_responses` and get the first `response`

Reply:
fixed